controller/controller_server/utils.py

```diff
@@ -65,8 +67,10 @@ def validate_csi_volume_capability(cap):
     logger.debug("csi volume capabilities validation finished.")
 
 
-def validate_csi_volume_capabilties(capabilities):
-    logger.debug("validating csi volume capabilities: {}".format(capabilities))
+def validate_csi_volume_capabilities(capabilities):
+    logger.debug("validating csi volume capabilities")
+    if not capabilities:
+        raise ValidationException(messages.capabilities_not_set_message)
     if len(capabilities) == 0:
         raise ValidationException(messages.capabilities_not_set_message)
```

Comment:
wouldn't `not capabilities` include `len(capabilities) == 0`?

Reply:
done